src/card.rs

```diff
@@ -91,18 +91,27 @@ impl<T: Client> iso7816::App for Card<T> {
 impl<T: Client, const C: usize, const R: usize> apdu_dispatch::App<C, R> for Card<T> {
     fn select(
         &mut self,
+        interface: apdu_dispatch::dispatch::Interface,
         command: &iso7816::Command<C>,
         reply: &mut heapless::Vec<u8, R>,
     ) -> Result<(), Status> {
+        use apdu_dispatch::dispatch::Interface;
```

Comment:
nit: Why not a top-level import?

Reply:
It would also need to be behind a the feature flag as `apdu_dispatch` is an optional dependency.